Makefile

```diff
@@ -125,3 +125,12 @@ destroy_folder: install
 .PHONY: binary
 binary:
 	podman run --pull=always --rm registry.ci.openshift.org/ci/rhcs-tf-bin:latest cat /root/terraform-provider-rhcs > ~/terraform-provider-rhcs && chmod +x ~/terraform-provider-rhcs
+
+.PHONY: e2e_testnew
+e2e_testnew: tools install
```

Comment:
I would address the naming of the new target right away in this PR. Having 'new' in the name will eventually not be true and isn't meaningful enough for people to understand what's the difference between this and ``e2e_test``
Maybe we should rename the ``e2e_test -> e2e_sanity_test`` or something alike? cc @bardielle

Reply:
@osherdp How about name the new one with to `e2e_test_full_cycle` ? I'd like not to change existing one with current CI still running.

Reply:
The CI doesn't use ``e2e_test``, so there's no concern about this target being used somewhere else